rover/src/owr_drive_controls/src/BoardControl.cpp

```diff
@@ -385,14 +391,20 @@ void BoardControl::run() {
                 firstRun = false;
             }
             //do joint calculations
+            // TODO Restore this before committing to master    
             swerveMotorVels swerveState = doVelTranslation(&(currentVel));
+            // swerveMotorVels swerveState = doCrabTranslation(&(currentVel));
+
             pwmFLW = frontLeftWheel.velToPWM(swerveState.frontLeftMotorV);
             pwmFRW = frontRightWheel.velToPWM(swerveState.frontRightMotorV);
             pwmBLW = backLeftWheel.velToPWM(swerveState.backLeftMotorV);
             pwmBRW = backRightWheel.velToPWM(swerveState.backRightMotorV);
             //TODO: this should actually be the angle from the encoders
             pwmFRS = frontRightSwerve.posToPWM(frontRightSwervePotMonitor.getPosition(), updateRateHZ);
             pwmFLS =  frontLeftSwerve.posToPWM(-frontLeftSwervePotMonitor.getPosition(), updateRateHZ);
+            pwmBRS = backRightSwerve.posToPWM(backRightSwervePotMonitor.getPosition(), updateRateHZ);
+            pwmBLS =  backLeftSwerve.posToPWM(-backLeftSwervePotMonitor.getPosition(), updateRateHZ);
+
             pwmLIDAR = lidar.velToPWM(updateRateHZ);
             
```

Comment:
Please revert changes to this file is only used in bluetounge

Reply:
Done